src/controllers/home.js

```diff
@@ -0,0 +1,3 @@
+import { testEnvironmentVariable } from '../../../Desktop/property-pro-api/src/settings';
```

Comment:
Check this path, it needs to be relative to your project folder, you don't have to go all the way to the desktop folder to import this settings file.